data/campaigns/Descent_Into_Darkness/achievements.cfg

```diff
@@ -1,12 +1,59 @@
 #textdomain wesnoth-did
+
+# achievements in DiD generally should be based around exploration and trying funny things in the campaign
+# these achievements shouldn't be so much about battle challenges, I want to give players another incentive to explore the little details
+# also the achievements are somewhat written as if from Malin's point of view which is why there's some weird and sometimes quirky flavor in them
 [achievement_group]
     display_name=_"Descent into Darkness"
     content_for=descent_into_darkness
     [achievement]
         id = "did_rat_eater"
-        name = _"Rat Eater"
-        description = _"Have a Ghoul eat way too many rats during Descent into Darkness's A Haunting in Winter."
-        icon="data/core/images/icons/potion_red_small.png"
+        name = _"A Big Hunk of Flesh"
+        description = _"Feed a Ghoul way too many rats."
+        icon="data/core/images/attacks/fangs-animal.png"
+        icon_completed="data/core/images/attacks/fangs-animal.png~SCALE(72,72)~BLIT("data/core/images/misc/achievement-frames/frame-8-acid.png",0,0)"
+    [/achievement]
+    [achievement]
+        id = "did_drogan"
+        name = _"Yer a Ghoul, Drogan"
```

Comment:
Should have a `# po:` comment, you can't expect all the translators to spot the reference.

However, I'd prefer not to have pop-culture references in the official campaigns.

Reply:
They don't have to translate it as the reference, it's just something for fun as the name of the achievement. I wouldn't mind if someone translating it construing it as Malin making fun of Drogan's uneducated country accent either.

Reply:
Fidelity is part of the translators' task:
* if a line is obscure, they will spend time understanding it, time that you should save them by adding the `# po:` comment. They aren't normal readers who can just skip past an odd line, they need to stop and work out what it says.
* if you want them to have additional artistic license for translating a specific line, that's something to put in the `# po:` comment.

Reply:
Okay good point I will add a po comment for them.